lore/end_users/use/README.md

```diff
@@ -73,3 +73,9 @@ what is known in git as a merge conflict.
 
 For information on how to resolve merge conflicts, refer to
 [Resolving a merge conflict on the command line](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/).
+
+**NOTE**: Repositories which require multiple continuous integration checks (
```

Comment:
Looks like there is an extra space at the end of this line? If not, the newline is causing a space to be rendered. See the [rendered file][1] for what I mean.

[1]:https://github.com/eggshell/eggshell.github.io/blob/03fc84f54da58a651c015eb977f4fec154316391/lore/end_users/use/README.md#competing-pull-requests